geoips/plugins/modules/procflows/single_source.py

```diff
@@ -1076,9 +1076,7 @@ def get_alg_xarray(
             LOG.interactive(
                 "  Interpolating data with interpolator '%s'...", interp_plugin.name
             )
-            final_xarray = interp_plugin(
-                area_def, alg_xarray, alg_xarray, **interp_args
-            )
+            final_xarray = interp_plugin(area_def, alg_xarray, None, **interp_args)
```

Comment:
Adding this comment so we don't merge this branch until v1.12.0-release has been merged.